db/migrate/20200420194305_set_defaults_caution_flags.rb

```diff
@@ -0,0 +1,19 @@
+class SetDefaultsCautionFlags < ActiveRecord::Migration[5.2]
+  def up
+    safety_assured do
```

Comment:
I think this can be done with `change_column_default` instead. https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/change_column_default

Then I think the `safety_assured` can come off too

Reply:
changed